Source/JavaScriptCore/wasm/WasmAirIRGenerator.cpp

```diff
@@ -505,6 +505,31 @@ class AirIRGenerator {
         auto airOp = airOpForSIMDReplaceLane(info);
         result = tmpForType(Types::V128);
         append(MoveVector, v, result);
+        switch (airOp) {
+        case VectorReplaceLaneFloat64:
+            if (isValidForm(airOp, Arg::Imm, Arg::Tmp, Arg::Tmp, Arg::Tmp)) {
+                append(airOp, Arg::imm(imm), s, result, tmpForType(Types::I64));
```

Comment:
These temps are both GP, so you can make both these cases the same. Also, since you are already checking that the form is valid, you don't need the switch.